tests/modules/emails/test_email.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring
+from app.modules.emails.models import RecordedEmail, EmailTypes, EmailRecord
+from app.extensions.email import Email
+from app.modules.site_settings.models import SiteSetting
+import uuid
+
+
+def _prep_sending(flask_app):
+    assert flask_app.config[
+        'TESTING'
+    ]  # this should only run in TESTING (non-sending) mode
+    flask_app.config['MAIL_OVERRIDE_RECIPIENTS'] = None
+
+    # this mocks using mailchimp, but wont send since we are in TESTING
+    SiteSetting.set('email_service', string='mailchimp')
+    SiteSetting.set('email_service_username', string='testing_' + str(uuid.uuid4()))
+    SiteSetting.set('email_service_password', string='testing_' + str(uuid.uuid4()))
+
+
+def _cleanup_sending():
+    SiteSetting.query.delete()
+
+
+def test_basic_send(flask_app):
+    _prep_sending(flask_app)
+    test_subject = 'test subject'
+    test_recipient = str(uuid.uuid4()) + '-test@example.com'
+    msg = RecordedEmail(test_subject)
+    msg.recipients = [test_recipient]
```

Comment:
Why aren't all three of these set in the REcordedEmail (or it's base classes) constructors rather than manually overwriting